src/NumericalAlgorithms/Spectral/Mesh.hpp

```diff
@@ -11,6 +11,7 @@
 
 #include "DataStructures/Index.hpp"
```

Comment:
- The commit message should be active: `Make Mesh option createable`
- There should be 2 commits. One for making `Spectral::Basis` option createable and one for making `Mesh` option createable. @nilsleiffischer can probably help you with splitting a commit in a video chat :)

Reply:
No worried @nilsdeppe we're dealing with a professional here ;)